pages/index.tsx

```diff
@@ -144,77 +275,284 @@ export default function Home() {
+              <div
+                className={`${styles.expandBar} ${styles.with} ${
+                  showWith && styles.active
+                }`}
+                onClick={() => toggleWith(!showWith)}
+              >
+                Expand&nbsp;<span className={styles.mobile}> to view</span>{" "}
+                Prisma Client query
+              </div>
+              <pre className={`${styles.code} ${!showWith && styles.hide}`}>
+                <code>{CODE_CACHE}</code>
+              </pre>
+            </Body>
+          </div>
+          <div className={styles.withoutAccelerate}>
+            <H3>
+              <img src="/clock.svg" /> Without Accelerate Cache
+            </H3>
+            <div className={styles.illustrationSection}>
+              <Animation
+                autoplay={false}
+                name="without-accelerate"
+                state={state}
+                className={styles.animation}
+                fit={reactCanvas.Fit.FitWidth}
+                badge={
+                  <span
+                    className={styles.badgeYellow}
+                    style={{ top: "-30px", left: "6%" }}
+                  >
+                    <img
+                      src="/locationDot.svg"
+                      color={defaultTheme.colors.orange[400]}
+                      width="8px"
+                      height="15px"
+                    />
+                    {history?.[0]?.location ? (
+                      <span>{history?.[0]?.location}</span>
+                    ) : null}
+                  </span>
+                }
+                badgeVirginia={
+                  <span className={`${styles.badgeYellow} ${styles.virginia}`}>
+                    <img
+                      src="/locationDot.svg"
+                      color={defaultTheme.colors.orange[400]}
+                      width="8px"
+                      height="15px"
+                    />
+                    <span>n.virginia</span>
+                  </span>
+                }
+              />
+            </div>
+            <Body className={styles.cardInfo}>
+              <div className={styles.numbers}>
+                <h4>{num.format((1_000 / withoutCacheLatency) * 60)}</h4>
+                <span>queries per minute</span>
+              </div>
+              <div className={styles.numbers}>
+                <h4>{ms.format(withoutCacheLatency)}</h4>
+                <span>latency</span>
+              </div>
+              <p>
+                The database query and its response need to travel to the
+                database in&nbsp;
+                <span className={styles.badgeYellow}>
+                  <img
+                    src="/locationDot.svg"
+                    color={defaultTheme.colors.orange[400]}
+                    width="8px"
+                    height="15px"
+                  />
+                  {history?.[0]?.location ? (
+                    <span>{history?.[0]?.location}</span>
```

Comment:
I think this location is wrong, it should be the database's right?

<img width="566" alt="image" src="https://github.com/prisma/accelerate-speed-test/assets/8246284/18c3fe5f-f8f9-4833-aa35-fc5a392e68b0">

Reply:
@ankur-arch can you double check on Tatiana's comment and resolve if need be? Thank you

Reply:
@meletj I missed this comment. Yes, this should be the database's location.